Systematic creation of a ROOT broadcaster in the info() conflicts with other servlets

clueware opened the pull request:
In two occurrences, the systematic creation of a "/\*" broadcaster has created conflits:
 - when integration Atmosphere in an existing application with servlet already mapped: the "/\*" servlet mapping conflicts with all other mappings, even when the Atmosphere servlet is mapped to a subpath ("/atmosphere" for example);
 - when activating the JGroups Atmosphere functionality for a @ManagedService, the created "/\*" conflicts with the JGroups broadcaster, generating a lot of warning from the latter saying the message is not aimed at the broadcaster path, but to "/\*"

jfarcand commented:
@clueware I kind of agree with the patch, but can you share the log where you are seeing the warning? We need to be really careful here as I don't want the situation where no `Broadcaster` are available for some application. Having a test case would help as well, but I understand the complexity. Thanks!

jfarcand commented:
@clueware OK I implemented a different fixe based on your patch. Please try 2.3.0-SNAPSHOT. Thanks!

clueware commented:
Sorry, didn't checked my notifications early enough. I'll try with 2.3.0-SNAPSHOT and send you a minimalist usecase. The setup is quite simple though: create a service using the ManageService annotation bound to "/path", bind the AtmosphereServlet to this same path, then add the atmosphere-jgroups artifact to the pom. As soon as a message is sent to the service, the error occurs.

clueware commented:
Switched to your latest master, it's OK: I don't have the message anymore. Switched back to 2.2.6 to check, and sure enough the message comes back without your patch:
AVERTISSEMENT: discarded message from different cluster "/*" (our cluster is "/path"). Sender was deimos-1215
